crates/re_data_ui/src/image.rs

```diff
@@ -28,6 +28,7 @@ impl EntityDataUi for re_types::components::TensorData {
         verbosity: UiVerbosity,
         entity_path: &re_log_types::EntityPath,
         query: &re_data_store::LatestAtQuery,
+        store: &re_data_store::DataStore,
```

Comment:
NOTE: this is a personal style preference, so feel free to ignore.

When deciding parameters order, I usually go with "environmental first, specific last". `ctx` is basically always the same, so it is always first. `ui` is likewise part of the environment (its where we happen to add widgets). I would therefore have put `store` before `verbosity`.

This helps keep the argument order somewhat consistent across the code base (ctx, ui, store, entity_path, component_path, time, …). Of course, there is a lot of room for interpretation here.

Also, it'd be a pain in the ass to change this now, so feel free not to change anything - I just wanted to spread what I consider a good practice :)

Reply:
Yeah, you've mentioned that before and I had the thought in my head while writing this code.  If I was adding both query and store, I would have been more inclined to add it right after ctx at the beginning. But I found it easier to keep query & store together as a pair for this.

Open to doing parameter order reshuffling in another refactor-only PR and moving them both to the front. I'm also inclined to introduce some kind of "DataQueryContext" that combines those things just to cut down on all the arguments, which might also eventually include things like caches and override stacks.